Dnd.Ddd.Services/Commands/Handlers/ChooseCharacterRaceCommandHandler.cs

```diff
@@ -30,10 +30,10 @@ public void Handle(ChooseCharacterRaceCommand command)
             Guard.With<CharacterNotFoundException>().Against(character is null, command.CharacterUiD);
             Guard.With<InvalidOperationException>()
                 .Against(
-                    !(character is CharacterDraft),
+                    character.IsCompleted(),
                     $"Attempting to change race on a completed character with UiD: {command.CharacterUiD}!");
```

Comment:
Again - we can use domain exception instead of system one:
```suggestion
Guard.With<InvalidCharacterStateException>().Against(character.IsCompleted(), character.UiD);
```

Reply:
changed